### PR TITLE
User with photos first part

### PR DESCRIPTION
**Pull request by anowakowski**

### CODE REVIEW

File: src/app/app.module.ts [hunk 1]
```diff
@@ -21,7 +22,9 @@ import { ErrorInterceptorProvider } from './_services/error.interceptor';
 import { RoutePathService } from './_services/RoutePath.service';
 import { AuthGuard } from './_guards/auth.guard';
 import { MemberEditComponent } from './members/member-edit/member-edit.component';
-import { MemberListComponent } from './members/member-list/member-list.component';
+//import { MemberListComponent } from './members/member-list/member-list.component';
```
Comment (mvein) [inline src/app/app.module.ts]: Why this line is commented?

File: src/app/app.module.ts [hunk 2]
```diff
@@ -38,21 +41,30 @@ export function tokenGetter() {
       DiscoverComponent,
       GroupComponent,
       MemberEditComponent,
-      MemberListComponent
+      //MemberListComponent,
```
Comment (mvein) [inline src/app/app.module.ts]: Why this line is commented?

File: src/app/app.module.ts (hunk 1 above)
Comment (anowakowski) [reply src/app/app.module.ts]: yes, its shoulbe remove from SPA

File: src/app/app.module.ts (hunk 2 above)
Comment (anowakowski) [reply src/app/app.module.ts]: yes, its shoulbe remove from SPA ;)

**Pull request closed without merging**